keyboards/yd68/readme.md

```diff
@@ -0,0 +1,15 @@
+# yd68
+
+[yd68](https://imgur.com/gallery/Ygo668L)
+
+A short description of the keyboard/project
+
+Keyboard Maintainer: [Izzy84075](https://github.com/izzy84075)  
+Hardware Supported: YD68/YD68v2
```

Comment:
There should be two spaces at the end of this line, so the file is rendered correctly.